packages/dev/test/TestUtils.ts

```diff
@@ -42,6 +42,9 @@ export async function startRelay (
   args.push('--relayHubAddress', relayHubAddress)
   const configFile = path.resolve(__dirname, './server-config.json')
   args.push('--config', configFile)
+  args.push('--stakeManagerAddress', stakeManager.address)
```

Comment:
why? the SM was always retrieved from the RH.

Reply:
seemed like the easiest way, why read it if can pass it?